dashboard/test/factories/factories.rb

```diff
@@ -254,44 +258,45 @@
       end
     end
 
-    trait :unmigrated_google_sso do
-      unmigrated_sso
-      provider 'google_oauth2'
-      oauth_refresh_token 'fake-oauth-refresh-token'
+    trait :unmigrated_clever_sso do
+      unmigrated_untrusted_email_sso
+      provider 'clever'
```

Comment:
Once you merge #23271, we should do a find-and-replace-all for these provider strings

Reply:
I've got mixed feelings about it on these attributes, since these are 'providers' not 'credential_types' - we've got a clear mapping for all the sso providers though, so maybe it doesn't matter.

Reply:
yeah, that's a good point. also remembering now that we're going to be deleting the provider column at some point. depending on how far in the future that deletion is going to happen, i feel less weird leaving these strings lying around for a short while